app/routes/administrative-units/administrative-unit/governing-bodies/index.js

```diff
@@ -0,0 +1,26 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+
+export default class AdministrativeUnitsAdministrativeUnitGoverningBodiesIndexRoute extends Route {
+  @service store;
+
+  async model() {
+    let { id: administrativeUnitId } = this.paramsFor(
+      'administrative-units.administrative-unit'
+    );
+
+    let administrativeUnit = await this.store.findRecord(
+      'administrative-unit',
+      administrativeUnitId,
+      {
+        reload: true,
+        include: 'governing-bodies.has-time-specializations',
```

Comment:
I would also add the `governing-bodies.classification` relationship since it's fetched async now.